### ISSUE TITLE
Develop installation instructions/README

### ISSUE THREAD

**Issue by gregorgorjanc** [HighlanderLab/tree_seq_pipeline#5]
We should write how to install all the required software, including snakemake, Python, phasing, mamba/conda environments with tsinfer, ... all versioned

For example, I ran this on my Mac to install some snakemake version

```
mamba create -c bioconda -c conda-forge -n snakemake snakemake-minimal                                                                                       

mamba activate snakemake   
```

**Comment by hannesbecher** [comment]
Yes, we should also export the environments that are used by the pipeline, like `conda list -n myCoustomEnv --explicit > myCoustomEnv.txt`. Can then save these inside the repo. For this one would have to go through the rules and see which ends are used.
